packages/astro/src/core/config/config.ts

```diff
@@ -130,6 +130,11 @@ function mergeCLIFlags(astroConfig: AstroUserConfig, flags: CLIFlags) {
 		// TODO: Come back here and refactor to remove this expected error.
 		astroConfig.server.host = flags.host;
 	}
+	if (typeof flags.open === 'boolean') {
+		// @ts-expect-error astroConfig.server may be a function, but TS doesn't like attaching properties to a function.
+		// TODO: Come back here and refactor to remove this expected error.
```

Comment:
I don't clearly understand what you said to leave the link.
Could you please give me some tips on how to avoid it? 
Sorry about it.
Because I copied the previous code, I don't know why the code had TODO.

Reply:
I usually do things like:
```js
// TODO: <LINK_TO_ISSUE>  <explanation>
```

But that's fine like this too, considering you were just copying an existing comment :)